src/components/Radio/Radio.test.tsx

```diff
@@ -0,0 +1,123 @@
+import { expect } from "chai";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import { stub } from "sinon";
+
+import Radio from "./Radio";
+
+describe("Radio Button", () => {
+  let container;
+  let changeHandler;
+  let clickHander;
+
+  before(() => {
+    clickHander = stub();
+    changeHandler = stub();
+    container = Enzyme.mount(
+      <Radio
+        id="inputID"
+        attributes={{ onClick: clickHander }}
+        onChange={changeHandler}
+        labelText="Test Label"
+        showLabel={false}
+      ></Radio>
+    );
+  });
+
+  it("Renders with appropriate 'aria-label' attribute and value when 'showLabel' prop is set to false", () => {
```

Comment:
👍 
Can you add a check to an existing test to show that when `showLabel` is set to `true`, that the `aria-label` attribute does not get rendered?

Reply:
Done!

Reply:
Done!